.github/workflows/test.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         ./install.sh
         pip install pynvim
         pip install pytest
+        wget https://github.com/Kitware/CMake/releases/download/v3.21.3/cmake-3.21.3-linux-x86_64.sh
```

Comment:
Tests should run in both Linux and Macos. These instructions seem too Linux-specific. So they should go into the `.ghwf-Linux.sh`. There should be similar instruction for Macos. Alternatively, the test should only run in Linux.
Would the cmake version from apt be suitable for the test?

Reply:
I was trying to install it here so it would work for both. I can just got the prefix path wrong--it should be `$HOME`. Apt works fine.

Reply:
Would it really work if it marked linux-x86_64?

Reply:
Is it one of those new ARM Macs? Is homebrew or some other package manager?

Reply:
I'm not sure what architecture they are. And I wouldn't suggest using brew because that would increase the test running time noticeably. Let's just test cmake in Linux only.